src/features/search/MedewerkerDetail.vue

```diff
@@ -0,0 +1,146 @@
+<template>
```

Comment:
I think this was copied code from an older implementation. So I'm ignoring these changes.

Reply:
That's correct